versions.yaml

```diff
@@ -264,9 +264,9 @@ externals:
     version: "ea56c1bfa7e912f39e270ffe738b19ac8113af3d"
```

Comment:
Probably need to bump this too (otherwise we'll use the old configs with the new images)